consensus/tests/state_live_sync.rs

```diff
@@ -605,8 +605,12 @@ async fn can_reset_chain_of_chunks() {
     // Respond with erroneous chunk and then produce and gossip a new block.
     test_chunk_reset(
         |mock_node| {
-            mock_node.set_chunk_handler(Some(|request, blockchain| {
-                let mut chunk = request.handle(blockchain);
+            mock_node.set_chunk_handler(Some(|peer_id, request, blockchain| {
+                let mut chunk = <RequestChunk as Handle<
```

Comment:
Do we really need the typed `handle`?

Reply:
Yes, type annotations are needed in this case